src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -106,7 +133,7 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
           <Text style={styles.contentHeader}>{t('points.history.title')}</Text>
         )}
         <BottomSheetSectionList
-          contentContainerStyle={styles.contentContainer}
+          contentContainerStyle={sections.length ? {} : styles.contentContainer}
```

Comment:
note that there was some weird behavior here that i couldn't quite figure out; having `flex: 1` breaks scrolling on the section list, however, _without_ `flex: 1`, styling was totally off on the error state. some googling indicates this _might_ be an android thing, but not totally sure.

Reply:
ah, this rings a bell but i can't remember when i ran into this. the scrolling views are 2 nested containers and `style` controls the outer one and `contentContainerStyle` controls the inner one. perhaps applying flex:1 to one of them makes a difference (but maybe not, what you've got here is unoffencive)